addons/web/static/src/views/fields/many2many_binary/many2many_binary_field.js

```diff
@@ -29,12 +30,13 @@ export class Many2ManyBinaryField extends Component {
         return this.props.record.fields[this.props.name].string;
     }
     get files() {
-        return this.props.record.data[this.props.name].records.map((record) => {
+        const files = this.props.record.data[this.props.name].records.map((record) => {
             return {
                 ...record.data,
                 id: record.resId,
             };
         });
+        return this.props.reverseOrder ? files.sort((f1, f2) => f2.id - f1.id) : files;
```

Comment:
Hello,
Maybe I'm missing something, but why we don't sort by `id` if `reverseOrder` is `false` ?
Thx

Reply:
Or even, why do we sort by id in any case? Shouldn't you simply reverse the files order, without looking at ids at all?